src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -1516,6 +1516,57 @@
               }
             }
           }
+        },
+        "[apiCalls.savedObjectsImport.total]": {
+          "type": "long"
+        },
+        "[apiCalls.savedObjectsImport.kibanaRequest.yes]": {
+          "type": "long"
+        },
+        "[apiCalls.savedObjectsImport.kibanaRequest.no]": {
+          "type": "long"
+        },
```

Comment:
@Bamieh @mindbat 

To make use of the new `incrementCounter` method changes, I had to flatten these fields:

https://github.com/elastic/kibana/blob/60c38209c47cb27709b035cbef3674917145a0f8/src/plugins/kibana_usage_collection/server/collectors/core/core_usage_collector.ts#L118-L134

The `scripts/telemetry_check.js` script made these changes to the JSON file accordingly. Notice the brackets in each key name.

Is this OK? Or should I change the fields to use some other delimiting character instead of `.`?

Reply:
Please disregard, I figured out the problem and fixed it in 787ee5e.